website/content/specification/computer-example.xml

```diff
@@ -149,6 +178,19 @@
                 <description>Ensures that each USB device has a unique UUID.</description>
                 <key-field target="@uuid"/>
             </is-unique>
+            <allowed-values target="(motherboard|motherboard/cpu|motherboard/memory|motherboard/expansion-card|usb-device)/prop[not(@ns) or @ns=('http://csrc.nist.gov/ns/metaschema/computer/example')]/@name">
+                <enum value="model">CPU model</enum>
+            </allowed-values>
+            <has-cardinality level="WARNING" target="(motherboard|motherboard/cpu|motherboard/memory|motherboard/expansion-card|usb-device)/prop[not(@ns) or @ns=('http://csrc.nist.gov/ns/metaschema/computer/example')]" min-occurs="1">
+                <remarks>
+                    <p>Warn if a model is not provided.</p>
```

Comment:
Minor, non-blocking: maybe we should align with the style of the remark below? It sounds like the expectation is to make the remark seem like a user-facing message, even if it is not one.

```suggestion
                    <p>It is invalid to not provide a CPU model.</p>
```

Reply:
Ignoring this based on your verbal feedback.